cucumber/api/features/authenticate.feature

```diff
@@ -21,6 +21,19 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
+  Scenario: Authenticate response should be encoded if asked
+    Then I can POST "/authn/cucumber/alice/authenticate?encoded=true" with plain text body ":cucumber:user:alice_api_key"
```

Comment:
this should be `When I POST` as this is not what you are testing. you want to verify that `WHEN` you send the request `THEN` the token is encoded

Reply:
I agree. But I used to main authentication test above as an example.. Should I change it too?

Reply:
it is a different test. in the test above we verify that we can use the api-key to authenticate so the `Then` is correct. here you test that the response is encoded.